src/index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+  </head>
+  <body>
+    <h1 class="human">Universal Declaration of Human Rights</h1>
+    <h2 class="article">Preamble</h2>
```

Comment:
for the future, article is not best name for title/headline
class names should not depend on content(about human)